bin/lambda-cfn-init.js

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
```

Comment:
Similar to https://github.com/mapbox/lambda-cfn/pull/51/files#r132271944, we can delete this file (yay).